packages/dataviews/src/filter-and-sort-data-view.ts

```diff
@@ -121,7 +121,10 @@ export function filterSortAndPaginate< Item >(
 					} );
 				} else if ( filter.operator === OPERATOR_IS ) {
 					filteredData = filteredData.filter( ( item ) => {
-						return filter.value === field.getValue( { item } );
+						return (
```

Comment:
@mcsf I remember we talked about this during the prototype phase, where you asked why we weren't using boolean values for `backup` and `protect`. I finally got to investigate this and this is the fix.

Reply:
thanks for following up!